ecs-init/ecs-init.go

```diff
@@ -31,6 +31,7 @@ const (
 	PRESTART = "pre-start"
 	START    = "start"
 	PRESTOP  = "pre-stop"
+	STOP     = "stop"
```

Comment:
You don't appear to have updated the manpage to reflect this change.

Reply:
Indeed! I'll update and clarify how these targets should be used in the context of upstart and systemd.